src/Nethermind/Nethermind.Analytics/AnalyticsPlugin.cs

```diff
@@ -42,28 +44,37 @@ public void Dispose()
         public Task Init(INethermindApi api)
         {
             _api = api;
-            _analyticsConfig = _api.Config<IAnalyticsConfig>();
+            _analyticsConfig = api.Config<IAnalyticsConfig>();
+
             IInitConfig initConfig = _api.Config<IInitConfig>();
-            if (initConfig.WebSocketsEnabled &&
-                (_analyticsConfig.PluginsEnabled ||
-                 _analyticsConfig.StreamBlocks ||
-                 _analyticsConfig.StreamTransactions))
-            {
-                AnalyticsWebSocketsModule webSocketsModule = new AnalyticsWebSocketsModule(_api.EthereumJsonSerializer);
-                _api.WebSocketsManager!.AddModule(webSocketsModule, true);
-                _api.Publishers.Add(webSocketsModule);
+            _isOn = initConfig.WebSocketsEnabled &&
+                    (_analyticsConfig.PluginsEnabled ||
+                     _analyticsConfig.StreamBlocks ||
+                     _analyticsConfig.StreamTransactions);
+            
```

Comment:
shouldn't we give some info when those options are off? Like when we log info about jsonRpc not enabled or smth